internal/Util.hpp

```diff
@@ -214,6 +216,21 @@ static bool loadMoFileStringsToArray(FILE* moFile,
 	return true;
 }
 
+static std::vector<std::string> buildMoFilePaths(const char *domain) {
+	std::string languages = getenv("LANGUAGE");
+	std::vector<std::string> paths {""};
+	std::string slash = "/";
+	size_t pos = 0;
+	do {
+		pos = languages.find(":");
+		std::string lang = languages.substr(0, pos);
+		if (lang.empty()) break;
+		paths.emplace_back(slash + lang + "/LC_MESSAGES/" + domain + ".mo");
+		languages.erase(0, pos + 1);
```

Comment:
I think you should keep track of the value of `pos` at each iteration instead of removing parts of the string  something like:

```cpp
while (pos != languages.size())
{
    const size_t colon = languages.find(':', pos);
    if (colon == std::string::npos)
        break;
    std::string lang = languages.substr(pos, colon - pos);
    path.emplace_back(…);
    pos = colon + 1;
}
```
This will result in less strings created.